WORKSPACE

```diff
@@ -1,12 +1,35 @@
-git_repository(
+http_archive(
     name = "io_bazel_rules_go",
-    remote = "https://github.com/bazelbuild/rules_go.git",
-    tag = "0.5.5",
+    sha256 = "441e560e947d8011f064bd7348d86940d6b6131ae7d7c4425a538e8d9f884274",
+    strip_prefix = "rules_go-c72631a220406c4fae276861ee286aaec82c5af2",
+    urls = ["https://github.com/bazelbuild/rules_go/archive/c72631a220406c4fae276861ee286aaec82c5af2.tar.gz"],
```

Comment:
Do they also have a corresponding release version or a tag along with the SHA? If there is a human-readable equivalent for this SHA, could you please add that as a comment? Because now, I don't know why this revision was used instead of a release and how to choose a new revision in the future.

Reply:
I don't believe so. I pulled these from k/k, and have added a comment to that effect.

Reply:
Top-level comment helps a lot. Thanks for adding it!